hemi/electrs/conn.go

```diff
@@ -62,6 +62,13 @@ func (c *clientConn) call(ctx context.Context, method string, params, result any
 	log.Tracef("call")
 	defer log.Tracef("call exit")
 
+	// Check context before acquiring lock
```

Comment:
This is the first thing that happens in the underlying code and thus not needed.